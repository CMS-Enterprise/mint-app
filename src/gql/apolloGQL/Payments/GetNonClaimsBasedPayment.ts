import { gql } from '@apollo/client';

export default gql(/* GraphQL */ `
  query GetNonClaimsBasedPayment($id: UUID!) {
    modelPlan(id: $id) {
      id
      modelName
      payments {
        id
        payType
        payClaims
        nonClaimsPayments
        nonClaimsPaymentsNote
        nonClaimsPaymentOther
        paymentCalculationOwner
        numberPaymentsPerPayCycle
        numberPaymentsPerPayCycleNote
        sharedSystemsInvolvedAdditionalClaimPayment
        sharedSystemsInvolvedAdditionalClaimPaymentNote
        planningToUseInnovationPaymentContractor
        planningToUseInnovationPaymentContractorNote
      }
      operationalNeeds {
        id
        modifiedDts
      }
    }
  }
`);
