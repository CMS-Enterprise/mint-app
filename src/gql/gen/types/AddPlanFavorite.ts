/* tslint:disable */
/* eslint-disable */
// @generated
// This file was automatically generated and should not be edited.

// ====================================================
// GraphQL mutation operation: AddPlanFavorite
// ====================================================

export interface AddPlanFavorite_addPlanFavorite {
  __typename: "PlanFavorite";
  modelPlanID: UUID;
  userID: UUID;
}

export interface AddPlanFavorite {
  addPlanFavorite: AddPlanFavorite_addPlanFavorite;
}

export interface AddPlanFavoriteVariables {
  modelPlanID: UUID;
}
