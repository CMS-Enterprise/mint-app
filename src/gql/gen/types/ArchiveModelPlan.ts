/* tslint:disable */
/* eslint-disable */
// @generated
// This file was automatically generated and should not be edited.

// ====================================================
// GraphQL mutation operation: ArchiveModelPlan
// ====================================================

export interface ArchiveModelPlan_updateModelPlan {
  __typename: "ModelPlan";
  id: UUID;
}

export interface ArchiveModelPlan {
  updateModelPlan: ArchiveModelPlan_updateModelPlan;
}

export interface ArchiveModelPlanVariables {
  id: UUID;
  archived: boolean;
}
