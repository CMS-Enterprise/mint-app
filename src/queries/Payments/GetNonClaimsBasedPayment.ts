import { gql } from '@apollo/client';

export default gql`
  query GetNonClaimsBasedPayment($id: UUID!) {
    modelPlan(id: $id) {
      id
      modelName
      payments {
        id
        payType
        nonClaimsPayments
        nonClaimsPaymentOther
        paymentCalculationOwner
        numberPaymentsPerPayCycle
        numberPaymentsPerPayCycleNote
        sharedSystemsInvolvedAdditionalClaimPayment
        sharedSystemsInvolvedAdditionalClaimPaymentNote
        planningToUseInnovationPaymentContractor
        planningToUseInnovationPaymentContractorNote
        fundingStructure
      }
    }
  }
`;
