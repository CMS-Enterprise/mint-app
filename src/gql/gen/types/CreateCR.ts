/* tslint:disable */
/* eslint-disable */
// @generated
// This file was automatically generated and should not be edited.

import { PlanCRCreateInput } from "./../../../types/graphql-global-types";

// ====================================================
// GraphQL mutation operation: CreateCR
// ====================================================

export interface CreateCR_createPlanCR {
  __typename: "PlanCR";
  id: UUID;
  modelPlanID: UUID;
  idNumber: string;
  dateInitiated: Time;
  dateImplemented: Time | null;
  title: string;
  note: string | null;
}

export interface CreateCR {
  createPlanCR: CreateCR_createPlanCR;
}

export interface CreateCRVariables {
  input: PlanCRCreateInput;
}
