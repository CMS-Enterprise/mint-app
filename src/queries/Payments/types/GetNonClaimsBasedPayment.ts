/* tslint:disable */
/* eslint-disable */
// @generated
// This file was automatically generated and should not be edited.

import { PayType, ClaimsBasedPayType, NonClaimsBasedPayType } from "./../../../types/graphql-global-types";

// ====================================================
// GraphQL query operation: GetNonClaimsBasedPayment
// ====================================================

export interface GetNonClaimsBasedPayment_modelPlan_payments {
  __typename: "PlanPayments";
  id: UUID;
  payType: PayType[];
  payClaims: ClaimsBasedPayType[];
  nonClaimsPayments: NonClaimsBasedPayType[];
  nonClaimsPaymentsNote: string | null;
  nonClaimsPaymentOther: string | null;
  paymentCalculationOwner: string | null;
  numberPaymentsPerPayCycle: string | null;
  numberPaymentsPerPayCycleNote: string | null;
  sharedSystemsInvolvedAdditionalClaimPayment: boolean | null;
  sharedSystemsInvolvedAdditionalClaimPaymentNote: string | null;
  planningToUseInnovationPaymentContractor: boolean | null;
  planningToUseInnovationPaymentContractorNote: string | null;
  fundingStructure: string | null;
}

export interface GetNonClaimsBasedPayment_modelPlan_operationalNeeds {
  __typename: "OperationalNeed";
  modifiedDts: Time | null;
}

export interface GetNonClaimsBasedPayment_modelPlan {
  __typename: "ModelPlan";
  id: UUID;
  modelName: string;
  payments: GetNonClaimsBasedPayment_modelPlan_payments;
  operationalNeeds: GetNonClaimsBasedPayment_modelPlan_operationalNeeds[];
}

export interface GetNonClaimsBasedPayment {
  modelPlan: GetNonClaimsBasedPayment_modelPlan;
}

export interface GetNonClaimsBasedPaymentVariables {
  id: UUID;
}
