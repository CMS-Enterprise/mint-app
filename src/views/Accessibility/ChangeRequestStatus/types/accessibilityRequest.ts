/* tslint:disable */
/* eslint-disable */
// @generated
// This file was automatically generated and should not be edited.

import { AccessibilityRequestStatus } from "./../../../../types/graphql-global-types";

// ====================================================
// GraphQL query operation: accessibilityRequest
// ====================================================

export interface accessibilityRequest_accessibilityRequest_statusRecord {
  __typename: "AccessibilityRequestStatusRecord";
  status: AccessibilityRequestStatus;
}

export interface accessibilityRequest_accessibilityRequest {
  __typename: "AccessibilityRequest";
  id: UUID;
  name: string;
  statusRecord: accessibilityRequest_accessibilityRequest_statusRecord;
}

export interface accessibilityRequest {
  accessibilityRequest: accessibilityRequest_accessibilityRequest | null;
}

export interface accessibilityRequestVariables {
  id: UUID;
}
