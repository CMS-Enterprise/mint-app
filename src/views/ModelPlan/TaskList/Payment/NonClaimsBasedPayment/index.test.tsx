import React from 'react';
import { MemoryRouter, Route } from 'react-router-dom';
import { MockedProvider } from '@apollo/client/testing';
import { render, screen, waitFor } from '@testing-library/react';

import GetNonClaimsBasedPayment from 'queries/Payments/GetNonClaimsBasedPayment';
import { GetNonClaimsBasedPayment_modelPlan_payments as GetNonClaimsBasedPaymentType } from 'queries/Payments/types/GetNonClaimsBasedPayment';
import {
  ClaimsBasedPayType,
  NonClaimsBasedPayType,
  PayType
} from 'types/graphql-global-types';

import NonClaimsBasedPayment from './index';

const mockData: GetNonClaimsBasedPaymentType = {
  __typename: 'PlanPayments',
  id: '123',
  payType: [PayType.CLAIMS_BASED_PAYMENTS],
  payClaims: [ClaimsBasedPayType.OTHER],
  nonClaimsPayments: [NonClaimsBasedPayType.OTHER],
  nonClaimsPaymentOther: 'Lorem Ipsum',
  paymentCalculationOwner: null,
  numberPaymentsPerPayCycle: null,
  numberPaymentsPerPayCycleNote: null,
  sharedSystemsInvolvedAdditionalClaimPayment: null,
  sharedSystemsInvolvedAdditionalClaimPaymentNote: null,
  planningToUseInnovationPaymentContractor: null,
  planningToUseInnovationPaymentContractorNote: null,
  fundingStructure: null
};

const paymentsMock = [
  {
    request: {
      query: GetNonClaimsBasedPayment,
      variables: { id: 'ce3405a0-3399-4e3a-88d7-3cfc613d2905' }
    },
    result: {
      data: {
        modelPlan: {
          id: 'ce3405a0-3399-4e3a-88d7-3cfc613d2905',
          modelName: 'My excellent plan that I just initiated',
          payments: mockData,
          operationalNeeds: [
            {
              modifiedDts: ''
            }
          ]
        }
      }
    }
  }
];

describe('Model Plan -- NonClaimsBasedPayment', () => {
  it('renders without errors', async () => {
    render(
      <MemoryRouter
        initialEntries={[
          '/models/ce3405a0-3399-4e3a-88d7-3cfc613d2905/task-list/payment/non-claims-based-payment'
        ]}
      >
        <MockedProvider mocks={paymentsMock} addTypename={false}>
          <Route path="/models/:modelID/task-list/payment/non-claims-based-payment">
            <NonClaimsBasedPayment />
          </Route>
        </MockedProvider>
      </MemoryRouter>
    );

    await waitFor(() => {
      expect(
        screen.getByTestId('payment-non-claims-based-payment-form')
      ).toBeInTheDocument();
    });
  });

  it('matches snapshot', async () => {
    const { asFragment } = render(
      <MemoryRouter
        initialEntries={[
          '/models/ce3405a0-3399-4e3a-88d7-3cfc613d2905/task-list/payment/non-claims-based-payment'
        ]}
      >
        <MockedProvider mocks={paymentsMock} addTypename={false}>
          <Route path="/models/:modelID/task-list/payment/non-claims-based-payment">
            <NonClaimsBasedPayment />
          </Route>
        </MockedProvider>
      </MemoryRouter>
    );

    await waitFor(() => {
      expect(
        screen.getByTestId('payment-non-claims-based-payment-form')
      ).toBeInTheDocument();
    });
    expect(asFragment()).toMatchSnapshot();
  });
});
