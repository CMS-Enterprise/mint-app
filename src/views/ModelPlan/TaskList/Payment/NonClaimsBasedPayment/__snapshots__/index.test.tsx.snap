// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Model Plan -- NonClaimsBasedPayment > matches snapshot 1`] = `
<DocumentFragment>
  <nav
    aria-label="Breadcrumbs"
    class="usa-breadcrumb usa-breadcrumb--wrap"
  >
    <ol
      class="usa-breadcrumb__list"
    >
      <li
        class="usa-breadcrumb__list-item"
      >
        <a
          class="usa-link usa-breadcrumb__link"
          href="/"
        >
          <span>
            Home
          </span>
        </a>
      </li>
      <li
        class="usa-breadcrumb__list-item"
      >
        <a
          class="usa-link usa-breadcrumb__link"
          href="/models/ce3405a0-3399-4e3a-88d7-3cfc613d2905/collaboration-area"
        >
          <span>
            Model collaboration area
          </span>
        </a>
      </li>
      <li
        class="usa-breadcrumb__list-item"
      >
        <a
          class="usa-link usa-breadcrumb__link"
          href="/models/ce3405a0-3399-4e3a-88d7-3cfc613d2905/collaboration-area/task-list"
        >
          <span>
            Model Plan task list
          </span>
        </a>
      </li>
      <li
        aria-current="page"
        class="usa-breadcrumb__list-item usa-current"
      >
        <span>
          Payment
        </span>
      </li>
    </ol>
  </nav>
  <h1
    aria-live="polite"
    class="mint-h1 margin-top-4 margin-bottom-2"
    tabindex="-1"
  >
    Payment
  </h1>
  <p
    class="margin-top-0 margin-bottom-1 font-body-lg"
    data-testid="model-plan-name"
  >
    for My excellent plan that I just initiated
  </p>
  <p
    class="margin-bottom-2 font-body-md line-height-sans-4"
  >
    If there's a question or field that is not applicable to your model or you don't currently know the answer, you may leave it blank. If you need help, ask a question using the link below.
  </p>
  <div
    class="padding-2 bg-primary-lighter"
  >
    <div
      class="display-flex"
      data-testid="ask-a-question"
    >
      <svg
        class="usa-icon text-primary margin-right-1"
        focusable="false"
        height="1em"
        role="img"
        viewBox="0 0 24 24"
        width="1em"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M20 2H4c-1.1 0-1.99.9-1.99 2L2 22l4-4h14c1.1 0 2-.9 2-2V4c0-1.1-.9-2-2-2zm-7 9h-2V5h2v6zm0 4h-2v-2h2v2z"
        />
      </svg>
      <button
        class="usa-button usa-button--unstyled"
        data-testid="ask-a-question-button"
        type="button"
      >
        Start a discussion
      </button>
    </div>
  </div>
  <div
    class="grid-container padding-left-0 padding-right-0"
    data-testid="gridContainer"
  >
    <div
      class="grid-row grid-gap"
      data-testid="grid"
    >
      <div
        class="desktop:grid-col-6"
        data-testid="grid"
      >
        <form
          action="#"
          class="margin-top-6"
          data-testid="payment-non-claims-based-payment-form"
        >
          <fieldset
            class="usa-fieldset"
            data-testid="fieldset"
          >
            <h3
              aria-live="polite"
              class="mint-h1 margin-bottom-3"
              tabindex="-1"
            >
              Non-claims-based payment questions
            </h3>
            <div
              class="usa-form-group margin-top-4"
              data-scroll="nonClaimsPayments"
            >
              <label
                class="usa-label"
                data-testid="label"
                for="payment-nonclaims-payments"
                id="label-payment-nonclaims-payments"
              >
                Select which non-claims-based payments will you pay.
              </label>
              <div>
                <div
                  class="easi-multiselect usa-combo-box margin-top-1 css-b62m3t-container"
                  id="payment-nonclaims-payments"
                >
                  <span
                    class="css-1f43avz-a11yText-A11yText"
                    id="react-select-3-live-region"
                  />
                  <span
                    aria-atomic="false"
                    aria-live="polite"
                    aria-relevant="additions text"
                    class="css-1f43avz-a11yText-A11yText"
                    role="log"
                  />
                  <div
                    class=" css-1vl89cy-control"
                  >
                    <div
                      class=" css-1fdsijx-ValueContainer"
                    >
                      <div
                        class=" css-1jqq78o-placeholder"
                        id="react-select-3-placeholder"
                      >
                        1 selected
                      </div>
                      <div
                        class=" css-qbdosj-Input"
                        data-value=""
                      >
                        <input
                          aria-activedescendant=""
                          aria-autocomplete="list"
                          aria-expanded="false"
                          aria-haspopup="true"
                          aria-labelledby="label-payment-nonclaims-payments"
                          autocapitalize="none"
                          autocomplete="off"
                          autocorrect="off"
                          class=""
                          id="react-select-3-input"
                          role="combobox"
                          spellcheck="false"
                          style="opacity: 1; width: 100%; grid-area: 1 / 2; min-width: 2px; border: 0px; margin: 0px; outline: 0; padding: 0px;"
                          tabindex="0"
                          type="text"
                          value=""
                        />
                      </div>
                    </div>
                    <div
                      class=" css-1hb7zxy-IndicatorsContainer"
                    >
                      <button
                        aria-label="Clear selection"
                        class="usa-button--unstyled"
                        id="clear-selection"
                        tabindex="0"
                        type="button"
                      >
                        <div
                          aria-hidden="true"
                          class=" css-1h8bgg-indicatorContainer"
                        >
                          <svg
                            aria-hidden="true"
                            class="css-tj5bde-Svg"
                            focusable="false"
                            height="20"
                            viewBox="0 0 20 20"
                            width="20"
                          >
                            <path
                              d="M14.348 14.849c-0.469 0.469-1.229 0.469-1.697 0l-2.651-3.030-2.651 3.029c-0.469 0.469-1.229 0.469-1.697 0-0.469-0.469-0.469-1.229 0-1.697l2.758-3.15-2.759-3.152c-0.469-0.469-0.469-1.228 0-1.697s1.228-0.469 1.697 0l2.652 3.031 2.651-3.031c0.469-0.469 1.228-0.469 1.697 0s0.469 1.229 0 1.697l-2.758 3.152 2.758 3.15c0.469 0.469 0.469 1.229 0 1.698z"
                            />
                          </svg>
                        </div>
                      </button>
                      <span
                        class=" css-16b81-indicatorSeparator"
                      />
                      <div
                        aria-hidden="true"
                        class=" css-gjng1h-indicatorContainer"
                      >
                        <svg
                          aria-hidden="true"
                          class="css-tj5bde-Svg"
                          focusable="false"
                          height="20"
                          viewBox="0 0 20 20"
                          width="20"
                        >
                          <path
                            d="M4.516 7.548c0.436-0.446 1.043-0.481 1.576 0l3.908 3.747 3.908-3.747c0.533-0.481 1.141-0.446 1.574 0 0.436 0.445 0.408 1.197 0 1.615-0.406 0.418-4.695 4.502-4.695 4.502-0.217 0.223-0.502 0.335-0.787 0.335s-0.57-0.112-0.789-0.335c0 0-4.287-4.084-4.695-4.502s-0.436-1.17 0-1.615z"
                          />
                        </svg>
                      </div>
                    </div>
                  </div>
                  <div>
                    <input
                      name="payment-nonclaims-payments"
                      type="hidden"
                      value="OTHER"
                    />
                  </div>
                </div>
                <div
                  class="easi-multiselect--selected"
                >
                  <h4
                    class="text-normal margin-bottom-1"
                  >
                    Selected non-claims-based payments
                  </h4>
                  <ul
                    class="usa-list--unstyled"
                    id="payment-nonclaims-payments-tags"
                  >
                    <li
                      class="margin-bottom-05 margin-right-05 display-inline-block"
                    >
                      <span
                        class="usa-tag easi-multiselect--tag padding-1 bg-primary-lighter text-ink display-inline-flex text-no-uppercase flex-align-center"
                        data-testid="multiselect-tag--Other"
                        id="selected-OTHER"
                      >
                        Other 
                        <svg
                          aria-label="Remove Other"
                          class="usa-icon margin-left-05"
                          focusable="false"
                          height="1em"
                          role="button"
                          tabindex="0"
                          viewBox="0 0 24 24"
                          width="1em"
                          xmlns="http://www.w3.org/2000/svg"
                        >
                          <path
                            d="M19 6.41 17.59 5 12 10.59 6.41 5 5 6.41 10.59 12 5 17.59 6.41 19 12 13.41 17.59 19 19 17.59 13.41 12z"
                          />
                        </svg>
                      </span>
                    </li>
                  </ul>
                </div>
              </div>
              <div
                class="usa-form-group"
                data-scroll="payment-nonclaims-payments-other"
              >
                <label
                  class="usa-label text-normal"
                  data-testid="label"
                  for="nonClaimsPaymentOther"
                >
                  Please describe the other non-claims-based payments you will pay.
                </label>
                <input
                  class="usa-input"
                  data-testid="payment-nonclaims-payments-other"
                  id="payment-nonclaims-payments-other"
                  name="nonClaimsPaymentOther"
                  type="text"
                  value="Lorem Ipsum"
                />
              </div>
              <div
                class="margin-top-3 margin-bottom-6"
              >
                <button
                  class="usa-button usa-button usa-button--unstyled"
                  data-testid="add-note-toggle"
                  type="button"
                >
                  <svg
                    aria-hidden="true"
                    class="usa-icon margin-right-1"
                    focusable="false"
                    height="1em"
                    role="img"
                    viewBox="0 0 24 24"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
                    />
                  </svg>
                  Add an additional note
                </button>
              </div>
            </div>
            <div
              class="usa-form-group"
              data-scroll="payment-nonclaims-payments-owner"
            >
              <label
                class="usa-label"
                data-testid="label"
                for="paymentCalculationOwner"
              >
                Who will calculate the non-claims based payments?
              </label>
              <input
                class="usa-input"
                data-testid="payment-nonclaims-payments-owner"
                id="payment-nonclaims-payments-owner"
                name="paymentCalculationOwner"
                type="text"
                value=""
              />
            </div>
            <div
              class="usa-form-group"
              data-scroll="payment-nonclaims-payments-paycycle"
            >
              <label
                class="usa-label"
                data-testid="label"
                for="numberPaymentsPerPayCycle"
              >
                Number of payments per payment cycle
              </label>
              <p
                class="text-base margin-y-1"
              >
                This only applies if you are making non-claims-based payments
              </p>
              <input
                class="usa-input"
                data-testid="payment-nonclaims-payments-paycycle"
                id="payment-nonclaims-payments-paycycle"
                name="numberPaymentsPerPayCycle"
                type="text"
                value=""
              />
              <div
                class="margin-top-3 margin-bottom-6"
              >
                <button
                  class="usa-button usa-button usa-button--unstyled"
                  data-testid="add-note-toggle"
                  type="button"
                >
                  <svg
                    aria-hidden="true"
                    class="usa-icon margin-right-1"
                    focusable="false"
                    height="1em"
                    role="img"
                    viewBox="0 0 24 24"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
                    />
                  </svg>
                  Add an additional note
                </button>
              </div>
            </div>
            <div
              class="usa-form-group margin-top-4"
              data-scroll="payment-nonclaims-shared-involvement"
            >
              <label
                class="usa-label maxw-none"
                data-testid="label"
                for="payment-nonclaims-shared-involvement"
              >
                Will the Shared Systems be involved for additional payment of claims?
              </label>
              <fieldset
                class="usa-fieldset"
                data-testid="fieldset"
              >
                <div
                  class="usa-radio"
                  data-testid="radio"
                >
                  <input
                    class="usa-radio__input"
                    data-testid="payment-nonclaims-shared-involvement-true"
                    id="payment-nonclaims-shared-involvement-true"
                    name="sharedSystemsInvolvedAdditionalClaimPayment"
                    type="radio"
                    value="TRUE"
                  />
                  <label
                    class="usa-radio__label"
                    for="payment-nonclaims-shared-involvement-true"
                  >
                    Yes
                  </label>
                </div>
                <div
                  class="usa-radio"
                  data-testid="radio"
                >
                  <input
                    class="usa-radio__input"
                    data-testid="payment-nonclaims-shared-involvement-false"
                    id="payment-nonclaims-shared-involvement-false"
                    name="sharedSystemsInvolvedAdditionalClaimPayment"
                    type="radio"
                    value="FALSE"
                  />
                  <label
                    class="usa-radio__label"
                    for="payment-nonclaims-shared-involvement-false"
                  >
                    No
                  </label>
                </div>
              </fieldset>
              <div
                class="margin-top-3 margin-bottom-6"
              >
                <button
                  class="usa-button usa-button usa-button--unstyled"
                  data-testid="add-note-toggle"
                  type="button"
                >
                  <svg
                    aria-hidden="true"
                    class="usa-icon margin-right-1"
                    focusable="false"
                    height="1em"
                    role="img"
                    viewBox="0 0 24 24"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
                    />
                  </svg>
                  Add an additional note
                </button>
              </div>
            </div>
            <div
              class="usa-form-group margin-top-4"
              data-scroll="payment-use-innovation-payment-contractor"
            >
              <label
                class="usa-label maxw-none"
                data-testid="label"
                for="payment-use-innovation-payment-contractor"
              >
                Are you planning to use the Innovation Payment Contractor?
              </label>
              <p
                class="text-base margin-y-1"
              >
                Note: If there will be 30+ payees we recommend a contractor.
              </p>
              <fieldset
                class="usa-fieldset"
                data-testid="fieldset"
              >
                <div
                  class="usa-radio"
                  data-testid="radio"
                >
                  <input
                    class="usa-radio__input"
                    data-testid="payment-use-innovation-payment-contractor-true"
                    id="payment-use-innovation-payment-contractor-true"
                    name="planningToUseInnovationPaymentContractor"
                    type="radio"
                    value="TRUE"
                  />
                  <label
                    class="usa-radio__label"
                    for="payment-use-innovation-payment-contractor-true"
                  >
                    Yes
                  </label>
                </div>
                <div
                  class="usa-radio"
                  data-testid="radio"
                >
                  <input
                    class="usa-radio__input"
                    data-testid="payment-use-innovation-payment-contractor-false"
                    id="payment-use-innovation-payment-contractor-false"
                    name="planningToUseInnovationPaymentContractor"
                    type="radio"
                    value="FALSE"
                  />
                  <label
                    class="usa-radio__label"
                    for="payment-use-innovation-payment-contractor-false"
                  >
                    No
                  </label>
                </div>
              </fieldset>
              <div
                class="margin-top-3 margin-bottom-6"
              >
                <button
                  class="usa-button usa-button usa-button--unstyled"
                  data-testid="add-note-toggle"
                  type="button"
                >
                  <svg
                    aria-hidden="true"
                    class="usa-icon margin-right-1"
                    focusable="false"
                    height="1em"
                    role="img"
                    viewBox="0 0 24 24"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
                    />
                  </svg>
                  Add an additional note
                </button>
              </div>
            </div>
            <div
              class="margin-top-6 margin-bottom-3"
            >
              <button
                class="usa-button usa-button usa-button--outline margin-bottom-1"
                data-testid="button"
                type="button"
              >
                Back
              </button>
              <button
                class="usa-button"
                data-testid="button"
                type="submit"
              >
                Next
              </button>
            </div>
            <button
              class="usa-button usa-button usa-button--unstyled"
              data-testid="button"
              type="button"
            >
              <svg
                aria-hidden="true"
                class="usa-icon margin-right-1"
                focusable="false"
                height="1em"
                role="img"
                viewBox="0 0 24 24"
                width="1em"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M20 11H7.83l5.59-5.59L12 4l-8 8 8 8 1.41-1.41L7.83 13H20v-2z"
                />
              </svg>
              Save and return to task list
            </button>
          </fieldset>
        </form>
      </div>
    </div>
  </div>
  <div
    class="mint-page-number margin-y-6"
  >
    <span
      class="mint-page-number__page-num"
    >
      Page 4 of 5
    </span>
  </div>
</DocumentFragment>
`;
